asreview/state/utils.py

```diff
@@ -122,11 +127,43 @@ def state_from_file(data_fp):
         logging.error(f"File {data_fp} does not exist, cannot create state.")
         return None
 
-    if not Path(data_fp).suffix in STATE_EXTENSIONS:
-        logging.error(f"file {data_fp} does not end with {STATE_EXTENSIONS}.")
-        return None
+    if Path(data_fp).suffix == ".asreview":
+        base_state = state_from_dot_asreview(data_fp)
+    else:
+        if not Path(data_fp).suffix in STATE_EXTENSIONS:
+            logging.error(f"file {data_fp} does not end with {STATE_EXTENSIONS}.")
+            return None
+        base_state = _get_state_class(data_fp)(state_fp=data_fp, read_only=True)
     state = {
         os.path.basename(os.path.normpath(data_fp)):
-        _get_state_class(data_fp)(state_fp=data_fp, read_only=True)
+        base_state
     }
     return state
+
+
+def state_from_dot_asreview(data_fp):
```

Comment:
`state_from_asreview_file`? I think we should prefix this with `read_` for consistency. But that requires changes to the other functions in this file as well. 

Please add this one to `asreview.state.__init__.py` (can you add `state_from_file ` and `states_from_dir` as well?)

Reply:
Please add the function to the documentation https://raw.githubusercontent.com/asreview/asreview/master/docs/source/reference.rst.

Reply:
I could change it to ```read_asreview_file``` and the others to ```read_state_from_file```, ```read_states_from_dir```, but then they will have to be changed everywhere I guess. Also in visualization for example. Or do you only want to change the name of ```state_from_asreview_file```? That would be easier.

Reply:
Oké. Clear. We will work on the deprecation in another PR.